src/buffer/out/ut_textbuffer/TextAttributeTests.cpp

```diff
@@ -80,9 +80,8 @@ void TextAttributeTests::TestRoundtripMetaBits()
         COMMON_LVB_UNDERSCORE
     };
 
-    for (int i = 0; i < ARRAYSIZE(metaFlags); ++i)
+    for (unsigned short flag : metaFlags)

```

Comment:
I had no idea that this just worked in c++, that's really cool.

Reply:
It does except the main problem is for some reason in the tracelogging .cpp, I had to unroll the changes because of what I think is a compiler bug because only in that file would the compiler emit an error.